Formula/clingo.rb

```diff
@@ -0,0 +1,22 @@
+class Clingo < Formula
+  desc "ASP system to ground and solve logic programs"
+  homepage "https://potassco.org/"
+  url "https://github.com/potassco/clingo/archive/v5.2.1.tar.gz"
+  sha256 "b486b0adbd0a2acdfe55856036039318844ccb1e3510f2796caf03c5ee455378"
+
+  depends_on "bison" => :build
```

Comment:
the system version of bison that ships with macOS isn't sufficient?

Reply:
https://github.com/potassco/clingo/blob/613e390a4d127ba2ab4e58de32dbc10077b7482b/INSTALL.md#development-dependencies says it needs 2.5 but I only had 2.3.

Reply:
Hmm, I tried the build and it compiles without bison installed but I wonder what happens if I don't add it.